audio/rsgain/Portfile

```
@@ -0,0 +1,45 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem                      1.0
+
+PortGroup                       github 1.0
+PortGroup                       cmake 1.1
+
+github.setup                    complexlogic rsgain 3.5.2 v
```

Also good to add the `revision` line:


```suggestion
github.setup                    complexlogic rsgain 3.5.2 v
revision                        0
```

`revision` gets incremented if ever we make changes to a port's built contents, but aren't changing the current version.